src/SFA.DAS.FAT.Domain/Courses/ProviderRating.cs

```diff
@@ -30,3 +30,16 @@ public enum EmployerProviderRating
     [Description("No employer reviews")]
     NotYetReviewed
 }
+
+public enum ProviderRating
```

Comment:
Cannot we use one single enum instead of three different ones? If the plan is to delete the other two after, can these be marked obsolete

Reply:
they want different words under the description, so this is what I came up with.  I guess I can use a generic description in the enum, then use logic to apply the needed description? I'll look into it.

Reply:
this is now done